pkgs/development/tools/documentation/gnome-doc-utils/default.nix

```diff
@@ -10,7 +10,14 @@ python2Packages.buildPythonApplication {
     sha256 = "19n4x25ndzngaciiyd8dd6s2mf9gv6nv3wv27ggns2smm7zkj1nb";
   };
 
+  buildInputs = [ libxslt pkgconfig ];
```

Comment:
Is there a particular reason for not treating `pkgconfig` as native? If it is used during the build, it probably should be native.

Reply:
Sorry, my bad, I was not attentive.